values-schema.yaml

```diff
@@ -588,11 +592,24 @@ definitions:
         $ref: '#/definitions/resource'
         description: Requested resources (guaranteed).
     title: Pod resources
-  runAsUser:
+  linuxUserIdRange:
+    type: object
```

Comment:
```suggestion
```

Reply:
`type` must be explicit, due to Ajv (validate script). Otherwise it will compain.